packages/vue-component/plugin/tag-handler.js

```diff
@@ -160,7 +160,7 @@ VueComponentTagHandler = class VueComponentTagHandler {
       }
 
       // Export
-      script = script.replace(jsExportDefaultReg, 'return');
+      script += `\nreturn require('${this.inputFile.getDisplayPath()}').default;`
```

Comment:
Sorry, I didn't explain well. I am asking if this influences dynamic imports on the client. Dynamic imports return promises, but here we are loading this in sync. But probably one can then dynamically load the whole vue component?